sdk/monitor/Azure.Monitor.Query/src/LogsQueryClientOptions.cs

```diff
@@ -52,5 +52,11 @@ internal string GetVersionString()
                 _ => throw new ArgumentException(@"Unknown version {_version}")
             };
         }
+
+        /// <summary>
+        /// Gets or sets the audience to use for authentication with Azure Active Directory (AAD). The audience is not considered when using a shared key.
```

Comment:
I've already completed the Azure AD to Entra ID rebranding in the Identity library. Let's update it here while you're in the file. I'll be opening a larger PR to catch all of the occurrences.
```suggestion
        /// Gets or sets the audience to use for authentication with Microsoft Entra ID. The audience is not considered when using a shared key.
```